### ISSUE TITLE
three-way comparison operator is showing up as "operator" instead of "operator<=>" in related symbols

### ISSUE THREAD

**Issue by scott-zhong** [doxygen/doxygen#11130]
**Describe the bug**
three-way comparison operator is showing up as "operator" instead of "operator<=>" in related symbols. See the testcase

```
#include <compare>

/**
 * X docs
 */
struct X
{
  /**
   * operator<=>(const struct X&, const struct X&)
   */
  friend std::strong_ordering operator<=>(const struct X&, const struct X&) = default;
};

/**
 * \relates X
 * operator<=>(const struct X&, const struct X&)
 */
std::strong_ordering operator<=>(const struct X&, char);
```

**Screenshots**
With Doxygen 1.11.0 - trunk (7e1860534d163a9d8882ce3ebb620719f74e75eb)

![image](https://github.com/user-attachments/assets/8dd497ad-941b-475a-8b6b-05fcf1adcd09)

The highlighted area shows "operator" instead of the expected "operator<=>"

**To Reproduce**
[testcase.zip](https://github.com/user-attachments/files/16955866/testcase.zip)

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #11132

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by scott-zhong** [comment]
@albert-github @doxygen I have verified the fix. Thank you!!!

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.13.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
